### DISCUSSION TITLE
Possibility to customize POI/Bookmarks icon

### DISCUSSION THREAD

**Discussion by Dret42** [organicmaps/organicmaps#4030] [Ideas]
Hi to all, and thanks for the great work made with Organic Maps!

I would like to suggest a little improvement for the POI/Bookmarks management:

 the possibility to customize icon for any single group of  POI/Bookmarks  to visually categorize them  bettter (even only a different color for each).

Thanks!

**Comment by biodranik** [comment]
Please follow #2795

**Comment by Bryum** [comment]
I'd like to add: not only custom icons for _groups_ of bookmarks would be nice, but for _individual_ bookmarks as well.

It's the one thing I'm missing since switching from OsmAnd (for speed reasons; old phone).

_Many_ thanks for this brilliant software!

**Comment by mdubbelm** [comment]
Adding to this: I'm a new user migrating from Google Maps. Over there, I have multiple lists with favourites. Each list represents a type of POI, so it's easy to recognize. It would be great if this feature would become available in OM. Love using it so far!
